dlrover/trainer/tests/torch/elastic_sampler_test.py

```diff
@@ -58,7 +58,7 @@ def test_checkpoint(self):
         )
         sampler.load_state_dict(sampler_state)
         val = next(iter(sampler))
-        self.assertEqual(val, 64)
+        self.assertEqual(val, 66)
```

Comment:
This should be caused by scaling. This unit-test tells us that data will be lost.